langwatch/src/components/TraceToPlaygroundLink.tsx

```diff
@@ -0,0 +1,70 @@
+import { Link } from "@chakra-ui/next-js";
+import {
+  Button,
+  Tooltip,
+  Flex,
+  Box,
+  type ButtonProps,
+  type PlacementWithLogical,
+} from "@chakra-ui/react";
+import { Image as ImageIcon } from "react-feather";
+import React from "react";
+
+// eslint-disable-next-line @typescript-eslint/no-namespace
+export namespace TraceToPlaygroundLink {
+  export interface Props {
+    projectSlug: string;
+    traceId: string;
+    spanId: string;
+    tooltipLabel?: string;
+    buttonLabel?: string;
+    tooltipGutterSize?: number;
+    tooltipPlacement?: PlacementWithLogical;
+    buttonVariant?: ButtonProps["variant"];
+  }
+}
+
+const DEFAULT_TOOLTIP_GUTTER_SIZE = 16;
+
+const DEFAULT_TOOLTIP_PLACEMENT: PlacementWithLogical = "right";
+
+const DEFAULT_BUTTON_VARIANT = "outline";
+
+export function TraceToPlaygroundLink({
+  projectSlug,
+  traceId,
+  spanId,
+  tooltipLabel,
+  buttonLabel,
+  tooltipGutterSize = DEFAULT_TOOLTIP_GUTTER_SIZE,
+  tooltipPlacement = DEFAULT_TOOLTIP_PLACEMENT,
+  buttonVariant = DEFAULT_BUTTON_VARIANT,
```

Comment:
this component is only ever used in a single place, so there is no reason to make it "customizable", let's avoid adding new props unless it's really needed later on, no trying to predict future usages.

YAGNI and KISS

Reply:
Sure. Those customizable constants are removed.